common/src/main/scala/explore/model/reusability.scala

```diff
@@ -25,6 +27,8 @@ object reusability {
   // TODO: Move to lucuma-ui.
   implicit def quantityReuse[N: Reusability, U]: Reusability[Quantity[N, U]] =
     Reusability.by(_.value)
+  implicit val magnitudeValueReuse: Reusability[MagnitudeValue]              = Reusability.byEq
```

Comment:
I thought we had these on lucuma-ui. If not we could move them there later on

Reply:
I now moved them to lucuma-ui.